packages/react-ui-core/src/Drawer/Drawer.js

```diff
@@ -72,42 +71,33 @@ export default class Drawer extends Component {
 
     return (
       <div
-        className={classnames(theme.Drawer, className)}
+        className={classnames(
+          theme.Drawer,
+          theme[this.state.visible ? 'Drawer-on' : 'Drawer-off'],
```

Comment:
You seem to use this.state.visible a lot. Maybe pull it out above?

Reply:
yup. good call